api_examples_final.py

```diff
@@ -0,0 +1,538 @@
+# coding: utf-8
+
+"""
+    Spectra Logic Corporation Software
+
+    StorCycle® REST API Copyright © 2020 Spectra Logic Corporation  # noqa: E501
+
+    The version of the OpenAPI document: 0.1
+    Contact: developer@spectralogic.com
+"""
+
+# Used for Python 2/3 compatibility
+from __future__ import absolute_import
+
+import os.path
+import urllib3
+import time
+
+# Used to show a "pretty" print of the API response payloads
+from pprint import pprint
+
+# Necessary imports for using the API
+import openapi_client
+from openapi_client.models.api_storage_location_nas import ApiStorageLocationNas  # noqa: E501
+from openapi_client.models.api_project_scan import ApiProjectScan  # noqa: E501
+from openapi_client.models.api_project_archive import ApiProjectArchive  # noqa: E501
+from openapi_client.models.api_project_restore import ApiProjectRestore  # noqa: E501
+from openapi_client.rest import ApiException
+
+# Suppress certificate warnings in output
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+########################################
+# Setup general variables. Individual project names, filters, etc must be changed within each example.
+# Note: Change these variables to appropriate local variables to run examples
+example_host = "https://localhost/openapi"
+example_password = "example_password"
+example_username = "example_username"
+email_on_complete = 'example@example.com'
+
+# This time will vary based on the number of files in the directory tree. Adjust the time to be long enough such that a
+# scan job finishes before the archive is attempted and the archive is finished before the restore is attempted.
+sleep_time_seconds = 10
+
+# Note: If the example directory tree is not created, these example variables will need to be changed.
+example_directory_root = "C:\\ApiPythonExampleData"
```

Comment:
might want to make this something like os.path.join(os.path.abspath(os.sep), "ApiPythonExampleData") to make this platform independent

os.path.abspath(os.sep) should return C:\\ on windows and / on linux